docker/readme.md

```diff
@@ -35,6 +35,15 @@ Then, to render a scene `CircleToSquare` in a file `test_scenes.py`, call
 $ docker exec -it --user="$(id -u):$(id -g)" my-manim-container manim test.py CircleToSquare -qm
 ```
 
+## Jupyterlab
+Another alternative is to use the `manimcommunity/manim:jupyterlab` image, which spins up a local webserver running
+JupyterLab in whose Python kernel manim is installed and can be accessed via the `%%manim` cell magic. To use this container,
+run
+```
+$ docker run -it -p 8888:88888 manimcommunity/manim:jupyterlab
+```
+and then follow the instructions in the terminal.
+
```

Comment:
nice!
What do you think about adding information about the jupyter notebook option as well?

Reply:
If you choose to do so, you can access the "classic" notebook interface from within jupyterlab, I don't want to add another image specifically for classic notebooks.

Reply:
okay!